cmd/jobobject-util/main.go

```diff
@@ -0,0 +1,225 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/Microsoft/hcsshim/internal/jobobject"
+	"github.com/Microsoft/hcsshim/internal/processorinfo"
+	"github.com/urfave/cli"
+	"golang.org/x/sys/windows"
+)
+
+const (
+	processorLimitFlag  = "processorLimit"
+	processorCountFlag  = "processorCount"
+	processorWeightFlag = "processorWeight"
+	memoryLimitInMBFlag = "memoryLimitInMB"
+	affinityFlag        = "affinity"
+
+	usage = `jobobject-util is a command line tool for getting and setting job object limits`
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "jobobject-util"
+	app.Commands = []cli.Command{
+		getJobObjectLimitsCommand,
+		setJobObjectLimitsCommand,
+	}
+	app.Usage = usage
+
+	if err := app.Run(os.Args); err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
+}
+
+var getJobObjectLimitsCommand = cli.Command{
+	Name:      "get",
+	Usage:     "gets the container's resource limits",
```

Comment:
I notice a lot of container-specific wording in this tool. If we're treating this as a "job object tool", I think we should aim for it to not itself have a concept of a "container", and just work at the job object layer instead.

Reply:
I updated it to remove all references to containers :)